.github/workflows/update-fixed-env.yaml

```
@@ -0,0 +1,39 @@
+name: Fixed Environment YAML Monitor
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'env/environment.yaml'
+
+jobs:
+  update_environment_fixed:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Setup micromamba
+      uses: mamba-org/setup-micromamba@v1
+      with:
+        micromamba-version: latest
+        environment-file: envs/environment.yaml
+        log-level: debug
+        init-shell: bash
+        cache-environment: true
+        cache-downloads: true
+
+    - name: Update environment.fixed.yaml
+      run: |
+        mamba env export --file envs/environment.fixed.yaml
```

Should be without builds
```suggestion
        mamba env export --file envs/environment.fixed.yaml  --no-builds
```